assets/dev/scss/frontend/frontend.scss

```diff
@@ -1,5 +1,32 @@
-// Frontend Lite
-@import "frontend-lite";
+@import "../global/global";
+@import "breakpoints/proxy";
+@import "../helpers/mixins";
 
-// Widgets
-@import "widgets";
```

Comment:
@rami-elementor What is the purpose of this PR? Are we now trying to optimise the default frontend CSS file, even without the "Improved CSS Loading"?

Reply:
@louiswol94 You are correct.

The task is to eliminate `frontend-lite.min.css`. Making `frontend.min.css` as a lite file.

All the pages should load a light version of the frontend file, and widgets styles should always load conditionally, whether the experiment is active or not.